// ngx-m-carousel/esm2020/lib/ngx-m-carousel.module.mjs
import { NgModule } from '@angular/core';
import { NgxMCarouselComponent } from './ngx-m-carousel.component';
import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
import * as i0 from "@angular/core";
export class NgxMCarouselModule {
}
NgxMCarouselModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.9", ngImport: i0, type: NgxMCarouselModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
NgxMCarouselModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.9", ngImport: i0, type: NgxMCarouselModule, declarations: [NgxMCarouselComponent], imports: [BrowserAnimationsModule], exports: [NgxMCarouselComponent] });
NgxMCarouselModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.9", ngImport: i0, type: NgxMCarouselModule, imports: [[
            BrowserAnimationsModule
        ]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.9", ngImport: i0, type: NgxMCarouselModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        NgxMCarouselComponent
                    ],
                    imports: [
                        BrowserAnimationsModule
                    ],
                    exports: [
                        NgxMCarouselComponent
                    ]
                }]
        }] });
